Reference/Querying/IPublishedContent/index -v7.md

```diff
@@ -0,0 +1,32 @@
+---
+versionFrom: 7.0.0
+needsV8Update: "true"
```

Comment:
```suggestion
```
This yaml tag is for docs pages that needs to be verified they work for v8. If we have a v8 version then it will no longer be needed 🙂